server/commands/admin.py

```diff
@@ -87,6 +91,46 @@ def ooc_cmd_help(client, arg):
                     f"No such command or submodule ({arg}) has been found in the help docs."
                 )
 
+def ooc_cmd_whitelist(client, arg):
+	"""
+	Send a request to activate whitelist
+	Usage: /whitelist <name>
+	"""
+	if not (client.server.config["whitelist"]):
+		raise ArgumentError("Whitelist is not enabled on this server!")
+	if client.is_wlisted == True:
+		raise ArgumentError("You have already been whitelisted!")
+	if len(arg) == 0:
+		raise ArgumentError("You must specify a discord username.")
+	disc_name = arg
+	client.discord_name = disc_name
+	get_pid = hex((randrange(1000000)) * 100)
+	client.player_id=get_pid[2:]
+	threading.Timer(randrange(5, 10), client.server.webhooks.whitelistrequest, args=(disc_name, client)).start()
+	client.send_ooc(f"Whitelist request sent! Your Player ID is {client.player_id}")
```

Comment:
Needs to be more descriptive what the next step should be. Ideally it should grab the exact channel name.
```suggestion
	client.send_ooc(f"Whitelist request sent! Your Player ID is '{client.player_id}'. Copy-paste this ID into the whitelist discord channel to complete authentication!")
```